source/process/proj_scaffold.md

```diff
@@ -26,5 +50,5 @@ The video below briefly describes each part of the structure. The video describe
 * Install globally with npm and use with every project (`sudo npm install -g @bhdirect/start-project`).
```

Comment:
This is outdated. It should say:
"Install via **toolkit** npm package and use with every project (`npm i -g @bhdirect/toolkit`)."